README.md

```diff
@@ -18,6 +18,62 @@ dev_appserver .
 
 See [CONTRIBUTING.md](/CONTRIBUTING.md) for more information on local development.
 
+## Using the data
+
+All test result data is public. There are two types of gzipped JSON data files we store.
```

Comment:
-> There are two types of gzipped JSON data files we store: test run summary files, and individual test result files.